.github/dependabot.yml

```diff
@@ -40,6 +40,22 @@ updates:
       - dependency-name: github.com/elastic/mito
     open-pull-requests-limit: 2
 
+  # Cloud providers' SDK dependencies
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "elastic/obs-ds-hosted-services"
+      - "elastic/obs-infraobs-integrations"
```

Comment:
since this is an automation task that will open pull request(s) (right?), does it make sense that we also add the corresponding labels to the PR, as e.g. [done above](https://github.com/elastic/beats/pull/39495/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28L8-L10):

```
    labels:
      - automation
      - dependabot
```

Reply:
Ouch, I missed the labels! Thank you for pointing this out. Added!